Use correct architecture identifier for 64 bit linux.

SDohle opened the pull request:
- [x] All [tests](https://github.com/jfrog/jfrog-azure-devops-extension#testing) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----
'amd64' is not (and never was) in the list of possible values: 'arm', 'arm64', 'ia32', 'mips', 'mipsel', 'ppc', 'ppc64', 's390', 's390x', 'x32', and 'x64'. (https://nodejs.org/docs/latest-v16.x/api/process.html#processarch)

SDohle committed:
Use correct architecture identifier for 64 bit linux.
'amd64' is not (and never was) in the list of possible values: 'arm', 'arm64', 'ia32', 'mips', 'mipsel', 'ppc', 'ppc64', 's390', 's390x', 'x32', and 'x64'. (https://nodejs.org/docs/latest-v16.x/api/process.html#processarch)

SDohle commented:
I have read the CLA Document and I hereby sign the CLA

eyalbe4 commented:
Thanks for this PR @SDohle.
So if *amd64* isn't supposed, why not remove it from the *switch case* completely?

SDohle commented:
@eyalbe4 To remain backwards compatible if somebody has hacked his environments. But I would prefer the removal, too.
So I will update the PR if not anybody votes for the current solution...

SDohle committed:
Remove 'amd64' case
'amd64' is not (and never was) in the list of possible values: 'arm', 'arm64', 'ia32', 'mips', 'mipsel', 'ppc', 'ppc64', 's390', 's390x', 'x32', and 'x64'. (https://nodejs.org/docs/latest-v16.x/api/process.html#processarch)
